osversion/platform_compat_windows_test.go

```diff
@@ -0,0 +1,64 @@
+package osversion
+
+import (
+	"testing"
+)
+
+const (
+	RS5Prefix         = "10.0.17763"
+	V21H2ServerPrefix = "10.0.20348"
+	V22H2Win11Prefix  = "10.0.22621"
+)
+
+// Test the platform compatibility of the different
+// OS Versions considering two ltsc container image
+// versions (ltsc2019, ltsc2022)
+func Test_PlatformCompat_HostWS2022(t *testing.T) {
+	for _, tc := range map[string]struct {
+		host      string
+		guestOs   string
+		shouldRun bool
+	}{
+		"WS2022Host_ltsc2022": {
```

Comment:
Can we put this in release order?

Reply:
You mean the older ones first? RS5, WS2022 and then Windows 11, right?

Reply:
Yes just to make it easier to read